### ISSUE TITLE
Consider Dual-Asset support for PoWFaucet

### ISSUE THREAD

**Issue by wsdt** [bobanetwork/boba_legacy#1270]
Currently the PoWFaucet seems only to support one asset at a time (ERC20 or native). 
For testnet we may also want to issue BOBA tokens (Boba ETH). 

But we also just may consider using the testnet faucet while using the PoWFaucet for mainnet: 
https://github.com/bobanetwork/boba/pull/1227

**Comment by wsdt** [comment]
@sk-enya already made significant progress on the faucet integration for the gateway, so we may as well just continue that. 

Suggestions welcome. 

For the time being @sk-enya do you want to ensure that you don't delete the changes you made to integrate the testnet faucet into the gateway? 

Everything else is TBD once the faucets are live on mainnet imho.

**Comment by InoMurko** [comment]
In terms of priorities - @thenursegreg42069 pls review this

- HC Boba tokens are irelevant and accessing BOBA on testnet chains because of this too, it's not being used - it can be addressed in a different way - telegram, discord, email and we can send them testnet boba no problem.
- access Boba ERC20 tokens on Boba ETH - nice to have, not immediate requirement
**- access ETH on mainnet is what is important and should be addressed right away.**

**Comment by wsdt** [comment]
Agree, didn't think about alternative approaches like Discord etc. even if we keep HybridCompute. 
Would possibly also keep infra & engineering costs lower.
